vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -59,6 +59,8 @@ def build(self):
                     maxDate=self.max,
                     persistence=True,
                     persistence_type="session",
+                    dropdownPosition='bottom-start',
+                    clearable=False,
```

Comment:
I'm still getting `... not enough values to unpack (expected 2, got 1)` exception when I click again on the "start date" from the UI after it's already selected. Here is the demo on how to recreate the bug: 

https://github.com/mckinsey/vizro/assets/108530920/0a4dfaac-904f-4184-81e0-1cc92932879e

QQ: Why did we decide to set `clearable=False` by default? 

Potential solution: 
We should define what is expected behaviour after the "start data" is clicked again (i.e. when the server receives `[None]`, instead of `[{start_date}, {end_date}]`).
 - Do we want to do nothing (probably just `raise dash.exceptions.PreventUpdate inside the `_filter_date_between` function would work here) or
 - We want to return an empty figure (return an empty `pd.Series` from the `_filter_date_between ` function). - This seems like a better solution to me. It's more intuitive in a case someone clears the DatePicker (by clicking on the "start date" again) and then selects another filter/parameter on the page.

Reply:
Generally, agree with @petar-qb 👍 🚀 

Could you guys just ensure the behaviour is consistent across the form components e.g. when I clear the Dropdown now - what's happening? https://vizro.mckinsey.com/relationship-analysis

It doesn't seem to do any of the two proposed solutions above 😄  Would be great to double-check 👍

Reply:
So, do we want to return the empty pd.Series in this case, and then check in a different pr the behaviour of other components? @huong-li-nguyen @petar-qb

Reply:
I think we should the behaviour of other components now and replicate the same behaviour here. The thing i would leave for a different PR is if we want to deviate from the existing approach and optimise it.

Reply:
Returning empty pd.Series in this case seems like the best solution since an empty vm.Dropdown and vm.Checklist as filter also causes that target figure doesn't contain any data.

Reply:
Just for my own tracking of what's happening now: 

- If you add the start-date but no end-date the figure does not get updated
- If you clear both inputs, then you get an empty figure